### PR TITLE
Create evaluation policy

### PR DESCRIPTION
**Pull request by SepsiLaszlo**

### CODE REVIEW

**Comment by OmTheTurtle** [comment]
I the test can you use single quotes instead of double quotes when not using interpolation?

**Review by falusi94 — COMMENTED** [review]
I leave some comments here, maybe it appears in a strange way, I started writing a review here before the other PR was merged. 

Regarding the policy testing I recommend the [pundit-matchers](https://github.com/chrisalley/pundit-matchers), these one liners make the behavior much clearer and leaves less possibility for errors.

**Comment by SepsiLaszlo** [comment]
I made the requested changes and refactored the test with pundit-matchers.
I think the `all_action`  abstraction could worth it. What do you think?

**Review by falusi94 — COMMENTED** [review]
Great, actually I haven't thought about the `all_action` abstraction, but I see potential in it. Though I'd recommend to split it to groups I suggested in changes. I feel like the `entry_request_actions` and the `point_request_actions` reflect to functionalities that don't belong this place and can be moved their own controllers/policies at some point.

File: spec/policies/evaluation_policy_spec.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      evaluation.point_request_status = Evaluation::REJECTED
+
+      is_expected.to permit_actions(%i[submit_point_request cancel_point_request])
+    }
+
+    it {
+      evaluation.entry_request_status = Evaluation::REJECTED
+
+      is_expected.to permit_actions(%i[submit_entry_request cancel_entry_request])
+    }
```
[the remark this answers is not in the document]
Comment (falusi94) [reply spec/policies/evaluation_policy_spec.rb]: True, then I'd recommend to do the following:

```ruby
    context 'when the point request is rejected' do
      before { evaluation.point_request_status = Evaluation::REJECTED }

      it { is_expected.to permit_actions(%i[submit_point_request cancel_point_request]) }
    end

    context 'when the entry request is rejected' do
      before { evaluation.entry_request_status = Evaluation::REJECTED }

      it { is_expected.to permit_actions(%i[submit_entry_request cancel_entry_request]) }
    end
```

The `is_expected` is intended to be used in a one-liner format.

[the remark this answers is not in the document]
Comment (SepsiLaszlo) [reply spec/policies/evaluation_policy_spec.rb]: I made the requested changes.

File: spec/policies/evaluation_policy_spec.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+RSpec.describe EvaluationPolicy, type: :policy do
+  subject { described_class.new(user, evaluation) }
+
+  let(:evaluation) { create(:evaluation) }
+  let(:all_action) { %i[show current table edit update] +
+                     %i[submit_point_request cancel_point_request] +
+                     %i[submit_entry_request cancel_entry_request] }
```
Comment (falusi94) [inline spec/policies/evaluation_policy_spec.rb:7]: Could you remove this?

**Review by falusi94 — APPROVED** [review]
🚀

File: spec/policies/evaluation_policy_spec.rb (hunk 2 above)
Comment (SepsiLaszlo) [reply spec/policies/evaluation_policy_spec.rb]: Done. 
I missed it.

**Pull request merged**